Dockerfile

```diff
@@ -1,3 +1,11 @@
+#
+# This is the official docker image that is used for production deployments of docker.
+#
+# It has the marathon startup script as entrypoint.
+#
+# It will reresolve all dependencies on every change (as opposed to Dockerfile.development)
+# but it ultimately results in a smaller docker image.
+#
 FROM java:8-jdk
 
 RUN apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
```

Comment:
I think that the resulting image will be even smaller if you merge this line with the RUN on line 19.